gcp/README.md

```diff
@@ -708,59 +708,20 @@ If this downtime will be disruptive to your work ("We're giving a demo for the
 Prime Minister at that time!"), please let me know immediately.
 ```
 
-#### External backups
+### External backups
 
-##### Setup.
+#### Sending backup outside of the organization.
 
-Create a Terragrunt definition like `gcp/live/prd/k8s/kube-system/backup-exporter/terraform.tfvars` inside the exekube project. This file contains 3 variables:
-   * `destination_bucket` - The destination GCS bucket, i.e "gs://gpii-backup-external-prd".
-   * `replica_count` - the number of CouchDB replicas that the cluster has. This is important for copying all the snapshots of the cluster at the same time.
-   * `schedule` - Follows the same format as a Cron Job. i.e: `*/10 * * * *` to execute the task every 10 minutes.
+The process is mostly executed by the `gcloud compute images export` command, which uses Cloud Build service, which uses [Daisy](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy) to create a VM, attach the images based on the snapshots to backup, create a file from those images and send the result files to the external storage.
 
-i.e:
-```
-# ↓ Module metadata
-
-terragrunt = {
-  terraform {
-    source = "/project/modules//backup-exporter"
-  }
+The destination buckets are created automatically by [Terraform code outside the organization](https://github.com/gpii-ops/gpii-infra/tree/master/common/modules/gcp-external-backup).
```

Comment:
I don't understand what you mean by "Terraform code outside the organization".

I think you mean something like: `The destination buckets are created automatically by [Terraform code](link). These buckets live in the testing organization (gpii2test) rather than the main organization (gpii) to keep them safe if a disaster affects the main organization.`